uaclient/security.py

```diff
@@ -562,6 +563,40 @@ def fix_security_issue_id(
     }
 
     if "CVE" in issue_id:
+        # Check livepatch status for CVE in fixes before checking CVE api
+        try:
+            status_stdout, _ = system.subp(
+                [
+                    "canonical-livepatch",
+                    "status",
+                    "--verbose",
+                    "--format=json",
+                ]
+            )
+        except exceptions.ProcessExecutionError:
+            pass
+        if status_stdout:
+            try:
+                parsed_patch = json.loads(status_stdout)["Status"][0][
+                    "Livepatch"
+                ]
+
+                if parsed_patch:
+                    fixes = parsed_patch.get("Fixes", [])
+                    if any(
+                        fix.Name is issue_id.lower() and fix.Patched
```

Comment:
`fix` is a dict so we need to access the values with `["key"]`. Also for string comparison we need to use `==` because `is` will only return true if the strings are the same object in memory (and therefore may return false even if the contents are the same)
```suggestion
                        fix["Name"] == issue_id.lower() and fix["Patched"]
```